modules/ROOT/pages/resource-usage.adoc

```diff
@@ -0,0 +1,28 @@
+= View Runtime Fabric Status and Resources
```

Comment:
To avoid confusion, I'd suggest we remove "Resources"

Reply:
Updated this - thank you!